apps/studio/vitest.config.ts

```diff
@@ -24,8 +24,6 @@ export default defineConfig({
   test: {
     globals: true,
     environment: 'jsdom', // TODO(kamil): This should be set per test via header in .tsx files only
-    include: [resolve(dirname, './**/*.test.{ts,tsx}')],
-    restoreMocks: true,
```

Comment:
this causes polyfills for matchMedia to be reset before tests are even run, so it has to be left to the default `false` for components with framer-motion to render